gulpfile.js

```diff
@@ -13,9 +13,8 @@ require('laravel-elixir-vue-2');
  |
  */
 
-elixir((mix) => {
+elixir(function(mix) {
```

Comment:
Actually - scrap what I just said. I've just been reviewing the docs which seems to use the syntax you've used. I'll merge the PR now.